schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(200)
```

Comment:
- [ ] I think here this is not the way for updating the table so please use the alter command if you want to update an existed table because this is already created so please make alter command to add a new column 🚀😊💪🏻